### PR TITLE
Added Microsoft eCDN 1st party solution

### PR DESCRIPTION
**Pull request by JoshP5**
I would recommend the Peer5 logo, which is being sunsetted following the company's acquisition by Microsoft in Aug 2021, should be removed from the initial image.

### CODE REVIEW

File: Teams/teams-live-events/what-are-teams-live-events.md [hunk 1]
```diff
@@ -91,13 +91,12 @@ The live event streaming platform is made up of the following pieces:
 
 ### Enterprise Content Delivery Network (eCDN)
 
-The goal of eCDN is to take the video content from the internet and distribute the content throughout your enterprise without impacting network performance. You can use one of the following certified eCDN partners to optimize your network for live events held within your organization:
+The goal of eCDN is to take the video content from the internet and distribute the content throughout your enterprise without impacting network performance. You can use either [Microsoft's first-party eCDN solution](/ecdn), or alternatively one of the following certified eCDN partners to optimize your network for live events held within your organization:
```
Comment (JohanFreelancer9) [inline Teams/teams-live-events/what-are-teams-live-events.md:94]:
```suggestion
The goal of eCDN is to take the video content from the internet and distribute the content throughout your enterprise without impacting network performance. You can use either [Microsoft first-party eCDN solution](/ecdn) or, alternatively, one of the following certified eCDN partners to optimize your network for live events held within your organization:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @JoshP5, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by JoshP5** [comment]
@MicrosoftHeidi Can you help us to merge this pull request? Thanks!

@PeerDiego

**Comment by PeerDiego** [comment]
#sign-off

**Comment by PeerDiego** [comment]
Hello @MicrosoftHeidi. Could we get your assistance getting this one merged as well? Thanks.

**Pull request merged**